internal/querynode/impl.go

```diff
@@ -1049,6 +1049,26 @@ func (node *QueryNode) SyncReplicaSegments(ctx context.Context, req *querypb.Syn
 	return &commonpb.Status{ErrorCode: commonpb.ErrorCode_Success}, nil
 }
 
+//ShowConfigurations returns the configurations of queryNode matching req.Pattern
+func (node *QueryNode) ShowConfigurations(ctx context.Context, req *internalpb.ShowConfigurationsRequest) (*internalpb.ShowConfigurationsResponse, error) {
+	if !node.isHealthy() {
+		log.Warn("QueryNode.ShowConfigurations failed",
+			zap.Int64("node_id", Params.QueryNodeCfg.GetNodeID()),
```

Comment:
nodeID

Reply:
log in other interface like getMetric all use UnderScoreCase, if we need change them all.